src/main/java/bhs/devilbotz/commands/DriveCommand.java

```diff
@@ -8,16 +9,16 @@
 public class DriveCommand extends CommandBase {
   private final DriveTrain drive;
   private final DoubleSupplier speed;
-  private final DoubleSupplier rotation;
+  private final DoubleSupplier rot;
 
-  private final SlewRateLimiter speedSlewRateLimiter = new SlewRateLimiter(3);
+  private final SlewRateLimiter speedSlewRateLimiter = new SlewRateLimiter(10);
 
-  private final SlewRateLimiter rotationSlewRateLimiter = new SlewRateLimiter(3);
+  private final SlewRateLimiter rotationSlewRateLimiter = new SlewRateLimiter(10);
```

Comment:
These should be defined in Constants.java

Reply:
Already done